tutorials/hana-clients-choose-hana-instance/hana-clients-choose-hana-instance.md

```diff
@@ -142,7 +139,7 @@ Congratulations! You have connected to SAP HANA Cloud and performed a few querie
 
 SAP provides a free streamlined version of SAP HANA that runs on developer laptops called [SAP HANA, express edition](https://www.sap.com/cmp/td/sap-hana-express-edition.html).
 
-SAP HANA runs on a few versions of Linux.  SAP HANA, express edition provides virtual machine images that can be run on Microsoft Windows, OS X and Linux machines as well as binary installs.  There are many different install options that are explained at [Try SAP HANA, express edition](https://developers.sap.com/topics/sap-hana.html).  The currently available version can be seen in the [release notes](https://developers.sap.com/topics/sap-hana.html#releaseNotes).  
+SAP HANA runs on a few versions of Linux.  SAP HANA, express edition provides virtual machine images that can be run on Microsoft Windows, macOS and Linux machines as well as binary installs.  There are many different install options that are explained at [Try SAP HANA, express edition](https://developers.sap.com/topics/sap-hana.html).  The currently available version can be seen in the [release notes](https://developers.sap.com/topics/sap-hana.html#releaseNotes).  
```

Comment:
remove one space before "SAP HANA", "There are many" and "The currently"

Reply:
It seems that HTML ignores the additional space. So there is no need to remove it.